charts/stardog/values.yaml

```diff
@@ -94,7 +94,7 @@ livenessProbe:
 # Stardog readiness probe settings. Consider increasing initialDelaySeconds when using nodes with
 # K8S nodes with limited CPUs. It is not generally required to modify periodSeconds and timeoutSeconds.
 readinessProbe:
-  initialDelaySeconds: 90
+  initialDelaySeconds: 15
```

Comment:
Does this work as default on EKS?

Reply:
Yes. It's the readiness probe. It will should work anywhere and once the pod is ready it'll start reporting ready when the probe starts passing.

Reply:
Sounds like yes from slack coversation